install.rst

```diff
@@ -18,49 +18,136 @@
 Installing Zeek
 ===============
 
-Prerequisites
+To run Zeek, grab our official Docker images, download our Linux binary
+packages, install via Homebrew on your Mac, use the ports collection on FreeBSD,
+or build Zeek yourself.
+
+Docker Images
 =============
 
-Before installing Zeek, you'll need to ensure that some dependencies
-are in place.
+We provide `official Docker images <https://hub.docker.com/u/zeekurity>`_ on the
+Docker Hub. These images are based on Debian 11 and feature a complete Zeek
+installation with ``zeek``, ``zkg``, and the Spicy toolchain, but are otherwise
+minimal to avoid bloat in derived images. For example, if you'd like to install
+Zeek plugins in those images, you'll need to install the needed compiler
+toolchain.
```

Comment:
The Spicy toolchain has limited functionality without a compiler, too, right? Is it worth mentioning it prominently if the interesting functionality doesn't quite work out of the box?

Reply:
@bbannier, how do you think we should word this?

Reply:
> @bbannier, how do you think we should word this?

We briefly had talked about this previously. Most of the time one just needs to install a handful of packets, but for wildly different reasons:
- `g++` is needed for Spicy JIT
- `cmake`, `make`: typical requirement of Spicy analyzer packages, e.g., the one generated with the default template
- `libpcap-dev` (provided by `libpcap0.8-dev`): dev dependency required to build with certain Zeek headers, e.g., `packet_analysis/Analyzer.h`.

When we last discussed this there was some preference for adding a script to the image setting up this toolchain, but looking back this feels too intrasparent to me given that it might not include the actual requirements of a given use case, so maybe it would be better to give a short explicit blurb in the docs, e.g.,

> We provide official Docker images on Docker Hub at https://hub.docker.com/u/zeekurity.
> The images run Debian and feature a complete Zeek installation with ``zeek``,
> ``zkg``, and the Spicy toolchain, but are otherwise minimal to avoid bloat in
> derived images. For example, if you'd like to install Zeek plugins in those
> images, you'll need to install the needed toolchain, typically `g++` for
> compilation, `cmake` and `make` as a build tool, and `libpcap-dev` to build
> against Zeek headers:
>
>     apt-get update
>     apt-get install --no-recommends g++ cmake make libpcap-dev

Reply:
Great, thanks. I recall the script idea and still like it, fwiw. I'll word in as you suggested.